lib/mongo/crypt/binding.rb

```diff
@@ -78,6 +78,13 @@ class Binding
         end
       end
 
+      # Minimum version of libmongocrypt required by this version of the driver.
+      # An attempt to use the driver with any previous version of libmongocrypt
+      # will cause a `LoadError`.
+      #
+      # @api private
+      MIN_LIBMONGOCRYPT_VERSION = Gem::Version.new("1.5.0.pre.alpha0")
```

Comment:
How does `1.5.0.alpha1` compare to `1.5.0.pre.alpha0`? Maybe this should be `1.5.0.alpha`.

Reply:
Done